File: javaDesde0/ejercicios1/ProyectoP1/Exponenciador.java
public class Exponenciador{

    public Exponenciador(){
    
    }
    
    public int potencialterado(int m , int n){
        
        int resultado = 1;
        
            for(int i = 1; i<=n; i++){
            
                resultado = resultado*m;
            
            }
        return resultado;
        
    }

    public int potenciaConRecursion(int m, int n){
    
        if(n==0){ return 1;
        }
        else{ return m * potenciaConRecursion(m,n-1);}
    }
}

File: ejercicios_Python_2/conversionPython/Exponenciador.py
def potencialterado(m: int, n: int):

    resultado:int = 1

    for i in range(n):
        resultado = resultado*m
        print(resultado)
    return resultado

def potenciadorRecursivo(m: int, n: int):

    if n==0:
        return
    else:

        return print(m *(m,n-1)) 

parametro1: int = potencialterado(3,6)
parametro2: int = potenciadorRecursivo(3,6)